train_diffusion_prior.py

```diff
@@ -125,6 +127,13 @@ def train(image_embed_dim,
     # Create save_path if it doesn't exist
     if not os.path.exists(save_path):
         os.makedirs(save_path)
+        
+    # Load pre-trained model from DPRIOR_PATH
+    if RESUME:
```

Comment:
That's assuming the default model parameters are used

It would be better to :
* Save the hp parameters (all the kwargs of diffusionprior) into:
  * Wandb
  * The checkpoint (as another dict besides state dict)

That way we would know the params both from wandb and from the checkpoint

Reply:
ok, gotcha. Let me fix that.

Reply:
Fixed. Please take a look.